docs/lockfile_generation.md

```diff
@@ -34,18 +30,22 @@ The Phylum CLI can generate a lockfile when it is given a manifest file.
 [go]: https://go.dev
 [cargo]: https://www.rust-lang.org
 
+:::tip
```

Comment:
What is this?

Reply:
The triple colon syntax is a [Docusaurus admonition](https://docusaurus.io/docs/markdown-features/admonitions)

Reply:
I ***really*** don't like using platform-specific syntax like this.

We've already migrated between documentation providers once and it was pretty simple since it's all just markdown. I don't want the next time to be more difficult.